README.md

```diff
@@ -51,6 +51,29 @@ Or install it yourself as:
 
 In the following sections, we explain the API provided by the gem and some basic usage.
 
+Many aspects of AR2DTO are [configurable for individual models](#setting-up-your-models); typically this is achieved by passing options to the has_dto method within a given model. Some aspects of AR2DTO are [configured globally](#global-configuration) for all models.
+
+### Global Configuration
+Global configuration options affect all threads and models where has_dto has been defined. A common place to put these settings is in a Rails initializer file such as `config/initializers/ar2dto.rb`.
+These settings are assigned directly on the `AR2DTO.configure` object.
+
+Configuration options are:
+ - default_excluded_attributes
+
+Syntax examples:
+
+```ruby
+  # config/initializers/ar2dto.rb
+
+  AR2DTO.configure do |config|
```

Comment:
can you also adjust https://github.com/santib/ar2dto/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97 to be consistent with your approach?

Reply:
Could you elaborate?

Reply:
the link is not working properly, but in the README it says

```ruby
# config/initializers/ar2dto.rb
AR2DTO.setup do |config|
  config.active_model_compliance = true
end
```

that needs to be adjusted, maybe just changing `setup` for `configure`